doc/langref.html.in

```diff
@@ -4798,8 +4824,8 @@ test "errdefer unwinding" {
       </p>
       {#header_open|Basics#}
       {#code_begin|test|test_unreachable#}
-// unreachable is used to assert that control flow will never happen upon a
-// particular location:
+// unreachable is used to assert that control flow will never reach a
+// particular place in the code.
```

Comment:
This feels like a bit of a needless reword to me.

Reply:
This was considered an "editing issue" by #1854

> Under "unreachable", in the two lines "// unreachable is used to
assert that control flow will never happen upon // particular location:"
I suggest that the latter part be replaced with "will never reach a //
particular place in the code."

I personally don't feel particularly strongly either way (my own personal preference might be to replace "happen upon" with "reach", but leave "location" instead of "place in the code").

Reply:
I agree with your personal preference, perhaps you should go with that?

Reply:
Done :)